redi-connect-infrastructure/README.md

```diff
@@ -0,0 +1,93 @@
+# redi-connect-infrastructure
+
+In this project we keep all the infrastructure redi-connect needs to run. We do that writing infrastructure as code using [terraform](https://www.terraform.io/).
+
+## Prerequisites
+
+The first time one executes terraform on an azure subscription they would have to create an azure container where they can store terraform state.
+
+
+This is the list of prerequisites required for the `connect` ci/cd pipeline:
+
+1. Before you run `terraform init` - Create the backend for azure.
+
+    resource:  https://docs.microsoft.com/en-us/azure/developer/terraform/store-state-in-azure-storage
+    
+    We do this using terraform. The tf code to provision the backend is in the `az-remote-backend-*` files.
+    When you `terraform aply` for the first time the backend will be provisioned for you in azure.
```

Comment:
To me this reads as comment about the effects of running `terraform apply`, than an instruction to run the command. Can we make it clearer?

Reply:
Can you make a suggestion so it makes more sense for both of us?

Reply:
Sure, here's a suggestion: https://gist.github.com/ericbolikowski/19327658846c9cd31f3e26e877891f89/revisions?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

I upper-cased Azure some places and the Prerequisites intro and first bullet

Reply:
Makes sense. Actually re-reading this made me realise that some points werent up to date. Will update the README. Apologies @ericbolikowski

Reply:
Thanks :-) No need to apologize!! We're doing a PR review and building a beautiful merge putting our brains together ❤️